flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/typeutils/AvroSerializer.java

```diff
@@ -163,8 +224,9 @@ public T copy(T from, T reuse) {
 
 	@Override
 	public void copy(DataInputView source, DataOutputView target) throws IOException {
-		T value = deserialize(source);
-		serialize(value, target);
+		// we do not have concurrency checks here, because serialize() and
+		// deserialize() do the checks and the current mechanism does not handle
```

Comment:
looks like the end of the comment is missing

Reply:
oh, right, fixing...